python/py-kaptan/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-kaptan
+version             0.5.10
+maintainers         nomaintainer
+
+categories-append   devel
+description         Configuration parser.
+long_description    ${description}
+
+platforms           darwin
+license             BSD
+
+homepage            https://github.com/emre/kaptan
+master_sites        pypi:k/${python.rootname}/
+distname            ${python.rootname}-${version}
+checksums           md5     d8c9eab27e915b9aae5b65fdb7b0e834 \
```

Comment:
md5 and sha1 checksums are not necessary - they have been proved to be not-so-secure, so using them is not useful.